plugins/ocm-backend/dist-dynamic/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-ocm-backend-dynamic",
-  "version": "4.0.6",
+  "version": "4.0.8",
```

Comment:
4.0.8 is already released...

https://www.npmjs.com/package/@janus-idp/backstage-plugin-ocm-backend?activeTab=versions

Perhaps we want 4.0.10 ?